docs/ballooning.md

```diff
@@ -263,3 +263,15 @@ cannot be enabled later by providing a `polling_interval` non-zero value.
 Furthermore, if the balloon was configured with statistics pre-boot through a
 non-zero `stats_polling_interval_s` value, the statistics cannot be disabled
 through a `polling_interval` value of zero post-boot.
+
+## Notes
+
+- Firecracker has no control around the speed of inflation or deflation, this is
+  dictated by the guest kernel. As a result, if the balloon is left inflated the
+  host could see serverly affected performance, it is therefore recommended to
+  only have the balloon inflated for shorter periods of time to reclaim memory.
+
+- The `deflate_on_oom` flag which allows the balloon to deflate in an out of
+  memory scenario, if it has yet to reach its target size will continue to try
+  and inflate even after releasing memory. This can lead to high CPU usage with
+  a high allocations and deallocations.
```

Comment:
Rworded a bit. Not sure I grasp it all.

```
The `deflate_on_oom` flag allows the balloon to deflate in an out-of-memory scenario. 
If the balloon has not yet reached its target size, it will continue trying to inflate even after releasing memory. This can lead to high CPU usage in workloads with a high frequency of memory allocations and deallocations.
```